src/admin/index.spec.js

```diff
@@ -67,6 +67,7 @@ describe('Admin', () => {
     const cluster = createCluster({
       requestTimeout: 1,
       enforceRequestTimeout: true,
+      requestTimeoutCheckInterval: 1,
```

Comment:
Rather than introduce a top level option that requires a lot of knowledge to understand, can't we calculate an appropriate number based on the request timeout value? I'd rather not bloat the API if possible.